packages/arb-token-bridge-ui/package.json

```diff
@@ -29,7 +29,7 @@
     "graphql": "^16.8.1",
     "lodash-es": "^4.17.21",
     "next": "^13.2.4",
-    "next-query-params": "^4.1.0",
+    "next-query-params": "^4.2.3",
```

Comment:
had to bump it as `NextAdapter` is throwing type error with the latest typescript version